js/main.js

```diff
@@ -1,44 +1,96 @@
+var entryLists = document.querySelector('.entry-list');
+var entryFormFeature = document.querySelector('.entry-form-feature');
+var entriesNavBar = document.querySelector('.entries-navbar');
+var entryView = document.querySelector('.entry-view');
+var noEntries = document.querySelector('.no-entries');
+
+photoUrl.addEventListener('input', function (event) {
+  if (photoUrl.value === '') {
+    imgSrc.setAttribute('src', './images/placeholder-image-square.jpg');
   } else {
-    $imgSrc.setAttribute('src', $photoUrl.value);
+    imgSrc.setAttribute('src', photoUrl.value);
   }
 });
 
-var $entryForm = document.querySelector('.entry-form'); // must assign '.entry-form' to a new variable first
-$entryForm.addEventListener('submit', newEntry); // listen for 'submit' events on the journal entry form. newEntry is made as a name for our function which will be stated in the next line.
-
-function newEntry(event) { // function is defined and named here called function newEvent(event) { }
-  event.preventDefault(); // event.preventDefault() will prevent the browser from automatically reloading the page with the form's values in the url.
-  var title = document.querySelector('.title').value; // select '.title' and assign it to a new variable.
-  var photoURL = document.querySelector('.photo').value; // select '.photo' and assign it to a new variable.
-  var notes = document.querySelector('.textbox').value; // select '.textbox' and assign it to a new variable.
-  var entryData = { // using JavaScript to gather user input from HTML forms using var entryData = { }.
-    // the title, photoURL, notes (the new variables) will be assigned to a new variable as a collective. This will be called entryData
-    title, // this is the variable we assigned to title.values and so on for the next lines 29 and 30.
+entryForm.addEventListener('submit', newEntry);
+function newEntry(event) {
+  event.preventDefault();
+  var title = document.querySelector('.title').value;
+  var photoURL = document.querySelector('.photo').value;
+  var notes = document.querySelector('.textbox').value;
+  var entryData = {
+    title,
     photoURL,
     notes
   };
-  entryData.entryId = data.nextEntryId;
+  data.entries.unshift(entryData);
   data.nextEntryId++; // incrementing
-  data.entries.unshift(entryData); // prepends. prepends means to put it in the beginning of something which is used as a .unshift();. With each data SAVED, it will be placed in the beginning of the object.
-  // localStorage.setItem('New Data', JSON.stringify(entryData)); // storing in localstorage by using localStorage.setItem(). Must use JSON.stringify() because localStorage only accepts data as a string.
-  $imgSrc.setAttribute('src', './images/placeholder-image-square.jpg');
-  $entryForm.reset(); // resets the entryForm once user clicks SAVE button
+  imgSrc.setAttribute('src', './images/placeholder-image-square.jpg');
+  entryForm.reset(); // resets the entryForm once user clicks SAVE button
 }
```

Comment:
Is the newly submitted entry visible as soon as it is submitted or do you have to refresh the page? It should be visible immediately after submission.